Traceback (most recent call last):
  File "C:\Users\edier\miniconda3\Lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\edier\miniconda3\Lib\site-packages\nbclient\client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\edier\AppData\Roaming\Python\Python311\site-packages\jupyter_core\utils\__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\edier\miniconda3\Lib\asyncio\base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "C:\Users\edier\miniconda3\Lib\site-packages\nbclient\client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "C:\Users\edier\miniconda3\Lib\site-packages\nbclient\client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\edier\miniconda3\Lib\site-packages\nbclient\client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
fp = "data/Barrios.shp"
db = gpd.read_file(fp)
------------------


[1;31m---------------------------------------------------------------------------[0m
[1;31mCPLE_OpenFailedError[0m                      Traceback (most recent call last)
File [1;32mfiona\ogrext.pyx:136[0m, in [0;36mfiona.ogrext.gdal_open_vector[1;34m()[0m

File [1;32mfiona\_err.pyx:291[0m, in [0;36mfiona._err.exc_wrap_pointer[1;34m()[0m

[1;31mCPLE_OpenFailedError[0m: data/Barrios.shp: No such file or directory

During handling of the above exception, another exception occurred:

[1;31mDriverError[0m                               Traceback (most recent call last)
Cell [1;32mIn[2], line 2[0m
[0;32m      1[0m fp [38;5;241m=[39m [38;5;124m"[39m[38;5;124mdata/Barrios.shp[39m[38;5;124m"[39m
[1;32m----> 2[0m db [38;5;241m=[39m [43mgpd[49m[38;5;241;43m.[39;49m[43mread_file[49m[43m([49m[43mfp[49m[43m)[49m

File [1;32m~\miniconda3\Lib\site-packages\geopandas\io\file.py:281[0m, in [0;36m_read_file[1;34m(filename, bbox, mask, rows, engine, **kwargs)[0m
[0;32m    278[0m     [38;5;28;01melse[39;00m:
[0;32m    279[0m         path_or_bytes [38;5;241m=[39m filename
[1;32m--> 281[0m     [38;5;28;01mreturn[39;00m [43m_read_file_fiona[49m[43m([49m
[0;32m    282[0m [43m        [49m[43mpath_or_bytes[49m[43m,[49m[43m [49m[43mfrom_bytes[49m[43m,[49m[43m [49m[43mbbox[49m[38;5;241;43m=[39;49m[43mbbox[49m[43m,[49m[43m [49m[43mmask[49m[38;5;241;43m=[39;49m[43mmask[49m[43m,[49m[43m [49m[43mrows[49m[38;5;241;43m=[39;49m[43mrows[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m
[0;32m    283[0m [43m    [49m[43m)[49m
[0;32m    285[0m [38;5;28;01melse[39;00m:
[0;32m    286[0m     [38;5;28;01mraise[39;00m [38;5;167;01mValueError[39;00m([38;5;124mf[39m[38;5;124m"[39m[38;5;124munknown engine [39m[38;5;124m'[39m[38;5;132;01m{[39;00mengine[38;5;132;01m}[39;00m[38;5;124m'[39m[38;5;124m"[39m)

File [1;32m~\miniconda3\Lib\site-packages\geopandas\io\file.py:322[0m, in [0;36m_read_file_fiona[1;34m(path_or_bytes, from_bytes, bbox, mask, rows, where, **kwargs)[0m
[0;32m    319[0m     reader [38;5;241m=[39m fiona[38;5;241m.[39mopen
[0;32m    321[0m [38;5;28;01mwith[39;00m fiona_env():
[1;32m--> 322[0m     [38;5;28;01mwith[39;00m [43mreader[49m[43m([49m[43mpath_or_bytes[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m [38;5;28;01mas[39;00m features:
[0;32m    323[0m         crs [38;5;241m=[39m features[38;5;241m.[39mcrs_wkt
[0;32m    324[0m         [38;5;66;03m# attempt to get EPSG code[39;00m

File [1;32m~\miniconda3\Lib\site-packages\fiona\env.py:457[0m, in [0;36mensure_env_with_credentials.<locals>.wrapper[1;34m(*args, **kwds)[0m
[0;32m    454[0m     session [38;5;241m=[39m DummySession()
[0;32m    456[0m [38;5;28;01mwith[39;00m env_ctor(session[38;5;241m=[39msession):
[1;32m--> 457[0m     [38;5;28;01mreturn[39;00m [43mf[49m[43m([49m[38;5;241;43m*[39;49m[43margs[49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwds[49m[43m)[49m

File [1;32m~\miniconda3\Lib\site-packages\fiona\__init__.py:308[0m, in [0;36mopen[1;34m(fp, mode, driver, schema, crs, encoding, layer, vfs, enabled_drivers, crs_wkt, allow_unsupported_drivers, **kwargs)[0m
[0;32m    305[0m     path [38;5;241m=[39m parse_path(fp)
[0;32m    307[0m [38;5;28;01mif[39;00m mode [38;5;129;01min[39;00m ([38;5;124m"[39m[38;5;124ma[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124mr[39m[38;5;124m"[39m):
[1;32m--> 308[0m     colxn [38;5;241m=[39m [43mCollection[49m[43m([49m
[0;32m    309[0m [43m        [49m[43mpath[49m[43m,[49m
[0;32m    310[0m [43m        [49m[43mmode[49m[43m,[49m
[0;32m    311[0m [43m        [49m[43mdriver[49m[38;5;241;43m=[39;49m[43mdriver[49m[43m,[49m
[0;32m    312[0m [43m        [49m[43mencoding[49m[38;5;241;43m=[39;49m[43mencoding[49m[43m,[49m
[0;32m    313[0m [43m        [49m[43mlayer[49m[38;5;241;43m=[39;49m[43mlayer[49m[43m,[49m
[0;32m    314[0m [43m        [49m[43menabled_drivers[49m[38;5;241;43m=[39;49m[43menabled_drivers[49m[43m,[49m
[0;32m    315[0m [43m        [49m[43mallow_unsupported_drivers[49m[38;5;241;43m=[39;49m[43mallow_unsupported_drivers[49m[43m,[49m
[0;32m    316[0m [43m        [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m
[0;32m    317[0m [43m    [49m[43m)[49m
[0;32m    318[0m [38;5;28;01melif[39;00m mode [38;5;241m==[39m [38;5;124m"[39m[38;5;124mw[39m[38;5;124m"[39m:
[0;32m    319[0m     colxn [38;5;241m=[39m Collection(
[0;32m    320[0m         path,
[0;32m    321[0m         mode,
[1;32m   (...)[0m
[0;32m    330[0m         [38;5;241m*[39m[38;5;241m*[39mkwargs
[0;32m    331[0m     )

File [1;32m~\miniconda3\Lib\site-packages\fiona\collection.py:243[0m, in [0;36mCollection.__init__[1;34m(self, path, mode, driver, schema, crs, encoding, layer, vsi, archive, enabled_drivers, crs_wkt, ignore_fields, ignore_geometry, include_fields, wkt_version, allow_unsupported_drivers, **kwargs)[0m
[0;32m    241[0m [38;5;28;01mif[39;00m [38;5;28mself[39m[38;5;241m.[39mmode [38;5;241m==[39m [38;5;124m"[39m[38;5;124mr[39m[38;5;124m"[39m:
[0;32m    242[0m     [38;5;28mself[39m[38;5;241m.[39msession [38;5;241m=[39m Session()
[1;32m--> 243[0m     [38;5;28;43mself[39;49m[38;5;241;43m.[39;49m[43msession[49m[38;5;241;43m.[39;49m[43mstart[49m[43m([49m[38;5;28;43mself[39;49m[43m,[49m[43m [49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[43mkwargs[49m[43m)[49m
[0;32m    244[0m [38;5;28;01melif[39;00m [38;5;28mself[39m[38;5;241m.[39mmode [38;5;129;01min[39;00m ([38;5;124m"[39m[38;5;124ma[39m[38;5;124m"[39m, [38;5;124m"[39m[38;5;124mw[39m[38;5;124m"[39m):
[0;32m    245[0m     [38;5;28mself[39m[38;5;241m.[39msession [38;5;241m=[39m WritingSession()

File [1;32mfiona\ogrext.pyx:588[0m, in [0;36mfiona.ogrext.Session.start[1;34m()[0m

File [1;32mfiona\ogrext.pyx:143[0m, in [0;36mfiona.ogrext.gdal_open_vector[1;34m()[0m

[1;31mDriverError[0m: data/Barrios.shp: No such file or directory

